dll/win32/shell32/folders/CDesktopFolder.cpp

```diff
@@ -448,12 +447,50 @@ HRESULT WINAPI CDesktopFolder::CompareIDs(LPARAM lParam, PCUIDLIST_RELATIVE pidl
 {
     bool bIsDesktopFolder1, bIsDesktopFolder2;
 
+    if (LOWORD(lParam) >= DESKTOPSHELLVIEWCOLUMNS)
+    {
```

Comment:
@HBelusca, Here is where a bad lParam (and indirectly iColumn) is handled now. Thanks.

Reply:
@HBelusca, Please see previous comment for where this is handled now. Thanks.